app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -54,10 +95,21 @@ angular.module('containerDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.
                     containerDashboardUtils.updateStatus($scope.objectStatus.services,   data.status.services);
                     containerDashboardUtils.updateStatus($scope.objectStatus.images,     data.status.images);
                     containerDashboardUtils.updateStatus($scope.objectStatus.routes,     data.status.routes);
+
+                    // Node utilization donut
+                    $scope.cpuUsageData = data.node_utilization.cpu;
+                    $scope.memoryUsageData = data.node_utilization.mem;
+                    $scope.utilizationLoadingDone = true;
+
+                    // Heatmaps
+                    $scope.nodeCpuUsage.data = data.heatmaps.nodeCpuUsage;
+                    $scope.nodeCpuUsage.loadingDone = true;
```

Comment:
> is this ever set to false? why is this variable needed? (and the rest of the similar vars)

Missed that but it still works because ```ng-if``` is ok with undefined variables. Will be fixed.